inode2prog.cpp

```diff
@@ -225,7 +225,9 @@ struct prg_node * findPID (unsigned long inode)
 		return node;
 	}
 
-	reread_mapping();
+	#ifndef __APPLE__
+		reread_mapping();
```

Comment:
Yes, you're right. But I think nethogs is still useful like this.
(And we might find a way to rewrite this function for OSX later.)